src/network/networkd-setlink.c

```diff
@@ -229,10 +229,14 @@ static int link_configure(
 
         log_link_debug(link, "Setting %s", set_link_operation_to_string(op));
 
-        if (IN_SET(op, SET_LINK_BOND, SET_LINK_CAN)) {
+        if (op == SET_LINK_BOND) {
                 r = sd_rtnl_message_new_link(link->manager->rtnl, &req, RTM_NEWLINK, link->master_ifindex);
                 if (r < 0)
                         return log_link_debug_errno(link, r, "Could not allocate RTM_NEWLINK message: %m");
+        } else if (op == SET_LINK_CAN) {
```

Comment:
Isn't this new block identical to the else block below?

Reply:
No, netlink commands are different: RTM_NEWLINK vs RTM_SETLINK.

Reply:
duh, and I checked 3 times